qa/zcash/performance-measurements.sh

```diff
@@ -65,6 +65,15 @@ function zcashd_valgrind_stop {
     cat valgrind.out
 }
 
+function extract_benchmark_data {
+    if [ ! -f "block-107134.tar.gz" ]; then
+        zcashd_stop
+        echo "block-107134.tar.gz not found."
```

Comment:
Once we have the file hosted somewhere, I plan to link to it here.

Or perhaps the script should just download the file if it isn't present? But if doing that, we'd need to somehow explain to the user what is going on, and what the additional file is, *without* polluting stdout (which is interpreted by CI).